notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -346,26 +346,6 @@
         "! gsutil mb -l $REGION -p $PROJECT_ID $BUCKET_URI"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #72.        y_train = tf.keras.utils.to_categorical(y_train, num_classes=num_species)
</pre><p>We are re-doing the categorization from above again. But using Keras. Any particular reason? Why not at least do it the same way?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>